multiqc/modules/rsem/rsem.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from Cutadapt """
+
+from __future__ import print_function
+import logging
+import re
+from distutils.version import StrictVersion
```

Comment:
I don't think you're using this package here? It was for pretty specific cutadapt task as I remember.